.editorconfig

```diff
@@ -9,3 +9,6 @@ insert_final_newline = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[fixtures/**.json]
+insert_final_newline = false
```

Comment:
I know this might be a headache with VSCode (and also sound picky), but it really depends on the editor you are using :sweat_smile: 

You can run `yarn run lint --write` before committing and that should do the trick, without modifying editorconfig.

Reply:
Oh I see, `yarn lint` calls `prettier` with `--ignore-path .gitignore` therefore doesn’t respect the fact that fixtures are ignored in `.prettierignore`. That’s… convoluted. Do you think we can somehow solve this at the CI / configuration layer?

Reply:
Probably the more correct approach would be to:
1. Remove the `JSON.parse`/`JSON.stringify` processing of Fragment Compositions’ definitions (in fact we don’t do it for Fragments’ configurations)
2. Remove `fixtures` from `.prettierignore`
3. Reformat the files through prettier so that `yarn lint` doesn’t frown

What do you think?

Reply:
I think you're right, I misunderstood the problem here. Also I think we should update the lint command to use .prettierignore instead of .gitignore (I am not sure why we added that, it looks like a but).

I tried to changed everything locally just to test but I still got formatting errors, maybe I am missing something. Could you give it a try?

![image](https://user-images.githubusercontent.com/800645/163165497-84868ca3-ca8a-43d9-b9fd-437cc70027f4.png)

Reply:
Removed that commit from the PR and applied what we discussed.